inlang/source-code/plugins/i18next/marketplace-manifest.json

```diff
@@ -2,6 +2,7 @@
 	"$schema": "https://inlang.com/schema/marketplace-manifest",
 	"id": "plugin.inlang.i18next",
 	"icon": "https://cdn.jsdelivr.net/gh/inlang/monorepo@main/inlang/source-code/plugins/i18next/assets/icon.png",
+	"displayImage": "https://cdn.jsdelivr.net/gh/inlang/monorepo@marketplace-enhancement/inlang/assets/marketplace/i18next-marketplace-cover.jpg",
```

Comment:
How about `bannerImage` instead of `displayImage`? maybe we introduce more images

Reply:
@samuelstroschein  maybe `coverImage` or similar because the main reason you will use it is not as a banner but more as a slight preview of what you offer, e.g. a thumbnail on Youtube?

Reply:
@floriandwt yeah sounds good.

Reply:
killer – lets go for `coverImage`!